### PR TITLE
[DGPFUS-2] - Develop a Product Page using HTML

### PR DESCRIPTION
**Pull request by Adezayn**
Products Page -PM
Products Page

### CODE REVIEW

**Comment by ToluEnny24** [comment]
@TundeOladejo

File: tests/package.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-  }
-}
+{
+  "dependencies": {
+    "@babel/core": "^7.15.8",
+    "chai": "^4.3.4",
+    "phantomcss": "^1.6.0",
+    "phantomjs-prebuilt": "^2.1.16",
+    "selenium-webdriver": "^4.0.0"
+  },
```
Comment (TundeOladejo) [inline tests/package.json:8]: You are not to make changes to this file. Please revert the changes.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]
